vendor/dawn.cmake

```diff
@@ -0,0 +1,100 @@
+# Dawn WebGPU Implementation Configuration
+
+if(TARGET mbgl-vendor-dawn)
+    return()
+endif()
+
+if(NOT MLN_WITH_WEBGPU)
+    return()
+endif()
+
+if(POLICY CMP0169)
+    cmake_policy(SET CMP0169 OLD)
+endif()
+
+set(MLN_DAWN_GIT_TAG "main" CACHE STRING "Git ref (branch, tag, or commit) used when fetching Dawn")
```

Comment:
I recommend pinning this so it doesn't just fail to build randomly in the future.